### DISCUSSION TITLE
use autoform to edit plugin alway throw error 'TypeError: Cannot read property 'properties' of undefined

### DISCUSSION THREAD

**Discussion by dan89farhan** [react-page/react-page#1001] [Q&A] [answered]
Describe the bug
use autoform to edit plugin alway throw error 'TypeError: Cannot read property 'properties' of undefined' when calling JSONSchemaBridge.getSubfields

To Reproduce
react: 17.0.2
@react-page/editor: 2.2.6
@react-page/plugns-video:2.2.6

step-1: add video plugin to page
step-2: edit video plugin

![image](https://user-images.githubusercontent.com/18419884/118520749-f37c9d00-b757-11eb-817f-f101e1f618a1.png)
@macrozone

**Comment by dan89farhan** [comment, marked as answer]
hi @PeterKottas and @macrozone, I solved the problem by changing the package.json file.
"uniforms-bridge-json-schema": "3.4.0"
Path: @react-page\editor\package.json
It resolved the error. please look into it. if you can patch the code

**Comment by macrozone** [comment]
that has been fixed in latest versions!

It now also workes with uniforms 3.5 or later.
